arm-resource-mngt-service/src/main/resources/application.properties

```diff
@@ -13,9 +13,9 @@ spring.cloud.config.uri=http://localhost:7000
 spring.profiles.active=dev
 
 ## MySQL
-spring.datasource.url=jdbc:mysql://localhost:3306/test1
```

Comment:
You don't need to checkin this file. These values are for your local setup

Reply:
removed